Editor/EditorAssetLoader.cs

```diff
@@ -22,18 +22,15 @@ public static void CreateSettingsAssets()
             AssetDatabase.Refresh();
             CreateAvatarConfigAssets();
             CreateAvatarLoaderSettings();
-            CreateReadyPlayerMeSettings();
+            CreatePartnerSubDomainSetting();
```

Comment:
SubDomain should be Subdomain. 

Kind of a side note but we could perhaps use PartnerName. or a PartnerSettings object with property Name. However not sure if there would ever be any other "Partner Settings" in future :D 

anyways PartnerSubdomainSettings is fine